tests/upgrades/test_repository.py

```diff
@@ -371,3 +372,152 @@ def test_post_scenario_sync_large_repo(self, target_sat, pre_upgrade_data):
+            'results'
+        ]
+        for content in all_content:
+            if content['name'] == custom_repo.name:
+                custom_content_label = content['label']
+            if content['repositories'][0]['id'] == rh_repo_id:
+                rh_content_label = content['label']
+        activation_key.content_override(
+            data={'content_overrides': [{'content_label': custom_content_label, 'value': '1'}]}
+        )
+        rhel_contenthost.register(
+            org=org, loc=None, activation_key=activation_key.name, target=target_sat
+        )
+        enabled = rhel_contenthost.execute("subscription-manager repos --list-enabled")
+        assert rh_content_label in enabled.stdout
+        assert custom_content_label not in enabled.stdout
+        sca_access = target_sat.execute(
+            f'echo "Organization.find({org.id}).simple_content_access?" | foreman-rake console'
+        )
+        assert 'false' in sca_access.stdout
+        sca_mode = target_sat.execute(
+            f'echo "Organization.find({org.id}).content_access_mode" | foreman-rake console'
+        )
+        assert 'entitlement' in sca_mode.stdout
+        sca_list = target_sat.execute(
+            f'echo "Organization.find({org.id}).content_access_mode_list" | foreman-rake console'
+        )
+        assert 'entitlement' in sca_list.stdout
+        assert rhel_contenthost.subscribed
+        save_test_data(
+            {
+                'rhel_client': rhel_contenthost.hostname,
+                'org_id': org.id,
+                'rh_content_label': rh_content_label,
+                'custom_content_label': custom_content_label,
+            }
+        )
+
+    @pytest.mark.parametrize('pre_upgrade_data', ['rhel7', 'rhel8', 'rhel9'], indirect=True)
+    @pytest.mark.post_upgrade(depend_on=test_pre_simple_content_access_only)
+    def test_post_simple_content_access_only(self, target_sat, pre_upgrade_data):
+        """Check that both the custom repository and the red hat repository are enabled
+        and that the host is in simple content access mode.
+
+        :id: postupgrade-6b418042-7bd7-40a6-9307-149614cc2672
+
+        :steps:
+            1. After satellite upgrade
+            2. Confirm host has both custom and red hat repository enabled
+            3. Confirm host is in simple content access mode
+
+        :expectedresults:
+            1. Content host is now in simple content access mode and custom repository
+            is enabled on host.
+        """
+        client_hostname = pre_upgrade_data.get('rhel_client')
+        org_id = pre_upgrade_data.get('org_id')
+        rh_content_label = pre_upgrade_data.get('rh_content_label')
+        custom_content_label = pre_upgrade_data.get('custom_content_label')
+        rhel_client = ContentHost.get_host_by_hostname(client_hostname)
+        enabled = rhel_client.execute("subscription-manager repos --list-enabled")
```

Comment:
making string format consistant
```suggestion
        enabled = rhel_client.execute('subscription-manager repos --list-enabled')
```